src/umpire/interface/c_fortran/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+##############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+set (umpire_interface_c_fortran_headers
+  wrapAllocator.h
+  wrapResourceManager.h
+  wrapUmpire.h
+  umpire.h
+  typesUmpire.h)
+
+set (umpire_interface_c_fortran_sources
+  utilUmpire.cpp
+  wrapAllocator.cpp
+  wrapResourceManager.cpp
+  wrapUmpire.cpp)
+
+if (ENABLE_FORTRAN)
+  set (umpire_interface_c_fortran_sources
+    ${umpire_interface_c_fortran_sources}
+    wrapfumpire.f)
+
+  set_source_files_properties(
+    wrapfumpire.f
+    PROPERTIES
+    Fortran_FORMAT FREE)
+endif ()
+    
+set_source_files_properties(
+  wrapResourceManager.cpp 
+  PROPERTIES 
+  COMPILE_FLAGS ${UMPIRE_DISABLE_DEPRECATED_WARNINGS_FLAG})
+
+set(umpire_interface_c_fortran_depends camp)
+blt_list_append( TO umpire_interface_c_fortran_depends ELEMENTS cuda_runtime IF ENABLE_CUDA )
+blt_list_append( TO umpire_interface_c_fortran_depends ELEMENTS hip_runtime IF ENABLE_HIP )
+blt_list_append( TO umpire_interface_c_fortran_depends ELEMENTS mpi IF ENABLE_MPI )
+
+blt_add_library(
+  NAME umpire_interface
+  HEADERS ${umpire_interface_c_fortran_headers}
+  SOURCES ${umpire_interface_c_fortran_sources}
+  DEPENDS_ON ${umpire_interface_c_fortran_depends}
+  OBJECT TRUE)
+
+target_include_directories(
+  umpire_interface
+  PUBLIC
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/src>
+  $<BUILD_INTERFACE:${CMAKE_BINARY_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+
+install(TARGETS
```

Comment:
I don't think you need this, since we are linking this into the uber libumpire. If you remove this you can also remove the `OBJECTS DESTINATION lib` line below.

Reply:
I got rid of line 53 above and the `OBJECTS DESTINATION lib` line.

Reply:
Update: that still caused errors only in the gcc8 docker build. Not sure why, but I reverted it back to what it was. Not sure how I should proceed here.